src/components/InventoryManager/EditListingPopup.js

```diff
@@ -315,31 +379,46 @@ export default function EditListingPopup({
           </Box>
         </Box>
       </DialogTitle>
-      <DialogContent className={classes.dialog}>
-        <form id="listing-form" onSubmit={onSubmit}>
-          <Grid spacing={3} container>
-            <Grid item xs={12}>
-              <EditListingBlock getListingRecord={getBasicInfo} name="Basic Information" index={1} />
-            </Grid>
-            <Grid item xs={12}>
-              <EditListingBlock getListingRecord={getUnitInfo} name="Units" index={2} />
-            </Grid>
-            <Grid item xs={12}>
-              <EditListingBlock getListingRecord={getPriceInfo} name="Price" index={3} />
-            </Grid>
-            <Grid item xs={12}>
-              <EditListingBlock getListingRecord={getDateInfo} name="Available Dates" index={4} />
+      <DialogContent className={classes.dialogContent}>
+        <Grid spacing={3} container>
+          <Grid item xs={12}>
+            <EditListingBlock getListingRecord={getBasicInfo} name="Basic Information" index={1} />
+          </Grid>
+          <Grid item xs={12}>
+            <EditListingBlock getListingRecord={getUnitInfo} name="Units" index={2} />
+          </Grid>
+          <Grid item xs={12}>
+            <EditListingBlock getListingRecord={getPriceInfo} name="Price" index={3} />
+          </Grid>
+          <Grid item xs={12}>
+            <EditListingBlock getListingRecord={getDateInfo} name="Available Dates" index={4} />
+          </Grid>
+          <Grid item xs={12}>
+            <EditListingBlock getListingRecord={getPhotoInfo} name="Photo" index={5} image />
+          </Grid>
+          <Grid container item xs={12} justify="flex-start" spacing={4}>
+            <Grid item>
+              <Button variant="outlined" size="large" type="submit" className={privatizedSelection === false ? classes.selectedButton : classes.publicizeButton} startIcon={<LockIcon />} onClick={() => onPrivatizeClick()}>
```

Comment:
For lines w/ multiple props that really long, could you put them 1 per line instead of all in 1 line for readability?